_fs/root-skel/etc/lighttpd.conf

```diff
@@ -85,28 +84,28 @@ accesslog.format = "%h %l %u %t \"%r\" %b %>s \"%{user-agent}i\" \"%{referer}i\"
 
 #######################################################################
 ##  Tuning/Performance
+## https://wiki.lighttpd.net/Docs_ResourceTuning
+## https://wiki.lighttpd.net/Docs_Performance
 #######################################################################
 server.event-handler = "poll"
-server.network-backend = "write"
```

Comment:
Maybe leave these as a comment? Low-level options may be very useful in the context of embedded systems. Have you checked that lighttpd's implicit choice works on Phoenix-RTOS?

Reply:
The comments is the links to the wiki with some explanations on the tunables and how they might be used and in what situations.  I personally disagree with leaving these commented-out, especially when "write" is the least efficient and should not be used unless others are not available.

However, I have made my suggestion and provided my justifications.  Please make your choice on what you will accept and edit this PR as you see fit so that you will accept it.

Reply:
> Have you checked that lighttpd's implicit choice works on Phoenix-RTOS?

lighttpd will use `sendfile()` if available and detected during compilation.  At runtime, if `sendfile()` fails (such as `ENOTSUP` or `EOPNOTSUPP` for a specific filesystem), then lighttpd will transparently fall back to `writev()`.  `write()` is for Windows lusers.

Reply:
If functions are detected at build-time and results checked at runtime, then removing this line should be fine. We aim to fix any issues regarding use of `sendfile()`/`writev()`/`write()` should they arise.